### PR TITLE
Confirm repo with initial files and GdQ's bio

### PR DESCRIPTION
**Pull request by malvikasharan**
Ask Gabi to add her details and confirm adding her image

### CODE REVIEW

**Comment by malvikasharan** [comment]
Hey Gabi, I have added some initial files: [README](https://github.com/jupytercon/2020-OpenSourceCommunities/blob/master/README.md), [CONTRIBUTING](https://github.com/jupytercon/2020-OpenSourceCommunities/blob/master/CONTRIBUTING.md), [LICENSE](https://github.com/jupytercon/2020-OpenSourceCommunities/blob/master/LICENSE.md), and [CoC](https://github.com/jupytercon/2020-OpenSourceCommunities/blob/master/CODE_OF_CONDUCT.md). Can you please also have a look at those and confirm you are happy with it.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ disabilities.
 
 If you experience or witness unacceptable behavior, or have any other concerns,
 please report it by contacting the project developers and maintainers:
-Malvika Sharan (email: malvikasharan@gmail)
-and Gabriela de Quieroz (contactkroz@gmail.com).
+Malvika Sharan (email: malvikasharan@gmail.com)
+and Gabriela de Quieroz (email: contactkroz@gmail.com).
```
Comment (gdequeiroz) [inline CODE_OF_CONDUCT.md]: Quieroz -> Queiroz

**Review by gdequeiroz — CHANGES_REQUESTED** [review]
LGTM

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md]: Done!

**Pull request merged**